### DISCUSSION TITLE
Plausible script is not loading from an Self Hosted instance

### DISCUSSION THREAD

**Discussion by mrdshyml** [plausible/analytics#2834] [Self-Hosted Support]
I have a self-installed instance on Digital Ocean. I am trying to install a Plausible script on my personal newsletter. I have used Google Tag Manager to install it. But Plausible is not receiving any data. 

On checking deeper, I saw my plausible script is not loading. 
<img width="378" alt="Screenshot 2023-04-07 150702" src="https://user-images.githubusercontent.com/80547432/230585607-9df580e6-ee7c-441b-86cb-91ae4cbb5970.png">

How can I fix that?

**Comment by ruslandoga** [comment]
👋 @mrdshyml 

Are there any errors in the console (the tab to the left of "Network" in your screenshot)? The script seems to be [accessible](https://plausible.indexdaily.in/js/app.js) at the URL in your screenshot, so I'd guess there is some CORS issue or similar.
